eng/pipelines/mirror-repos.yml

```diff
@@ -11,45 +11,55 @@ jobs:
     displayName: Sync public repos
 
     steps:
+    - checkout: none
```

Comment:
We will need to checkout once we start using script files from the tools repo.

Reply:
Actually during my test, the checkout failed once...